macros/scripts/events.asm

```diff
@@ -242,28 +242,6 @@ ResetEvents: MACRO
 ENDM
 
 
-;\1 = event index
-;\2 = number of bytes away from the base address (optional, for matching the ROM)
-dbEventFlagBit: MACRO
-	IF _NARG > 1
-		db ((\1) % 8) + ((\2) * 8)
-	ELSE
-		db ((\1) % 8)
-	ENDC
-ENDM
-
-
-;\1 = event index
-;\2 = number of bytes away from the base address (optional, for matching the ROM)
```

Comment:
Hah, I was just thinking yesterday how this comment - `(optional, for matching the ROM)` - is wrong anyway. It's certainly required for trainer view distance to work correctly.